### PR TITLE
Feature/issue 829 parallelization error

### PR DESCRIPTION
**Pull request by l-1squared**

### CODE REVIEW

File: jgiven-core/src/main/java/com/tngtech/jgiven/report/model/ReportModel.java [hunk 1]
```diff
@@ -38,161 +36,161 @@
...
+    public void setScenarios(List<ScenarioModel> scenarios) {
         this.scenarios = scenarios;
     }
 
     public String getPackageName() {
-        int index = this.className.lastIndexOf( '.' );
-        if( index == -1 ) {
+        int index = this.className.lastIndexOf('.');
+        if (index == -1) {
             return "";
```
Comment (fudler) [inline jgiven-core/src/main/java/com/tngtech/jgiven/report/model/ReportModel.java:111]: While refactoring a bit one could maybe use StringUtils here? :)

File: jgiven-junit5/src/test/java/com/tngtech/jgiven/junit5/test/concurrency/JUnit5InheritedParallelizationTest.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+    public static IntStream iterationProvider() {
+        return IntStream.range(0, 100);
+    }
+
+    @ParameterizedTest
+    @MethodSource("iterationProvider")
+    void firstTest(int i) {
+        given().a_thread_local_scenario_state();
+        when().the_state_on_this_thread_is_set_to("I am the greatest" + i);
```
Comment (fudler) [inline jgiven-junit5/src/test/java/com/tngtech/jgiven/junit5/test/concurrency/JUnit5InheritedParallelizationTest.java:23]: For convenience, probably there should be some additional space after greatest. The missing space somehow nudges me :D

File: jgiven-testng/src/test/java/com/tngtech/jgiven/testng/concurrency/TestNgInheritedParallelizationTest.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+import org.testng.annotations.Test;
+
+@Test(singleThreaded = false )
+@Ignore("root cause of error not yet fixed")
+public class TestNgInheritedParallelizationTest
+    extends ScenarioTest<Stages.ParallelGivenStage, Stages.ParallelWhenStage, Stages.ParallelThenStage> {
+
+    @Test
+    @Pending
+    public void fistTest() {
```
Comment (fudler) [inline jgiven-testng/src/test/java/com/tngtech/jgiven/testng/concurrency/TestNgInheritedParallelizationTest.java:15]: Typo

**Review by fudler — CHANGES_REQUESTED** [review]

**Comment by l-1squared** [comment]
Closing because this PR it was supposed to be #842.
The mergeable JUnit5 changes are included there.

**Pull request closed without merging**